### PR TITLE
Solve duplicates

### PR DESCRIPTION
**Pull request by niconoe**
Hey, as a follow-up to #41, this script removes duplicates into the existing `taxon` table at the beginning of the `transform_db` process.

A few considerations:

- It relies on a new configuration file (`deduplicate_taxon_config.json`), to specify for each duplicate which row should be preserved and which one should be replaced. The BIM team should carefully review this file prior to deployment (check that the choices in it are indeed correct and complete => recently created duplicates?)
- the script can be run multiple times, but it will only have effect once, and those effects are destructive (removing rows). This is different than the previous scripts, but I think this is acceptable (backups prior to migration are a must anyway, and trying to solve that would add a lot of fragility and complexity)

### CODE REVIEW

File: scripts/deduplicate_taxon.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# value: new taxon_id (to replace the other one)
+import os
+
+from helpers import get_database_connection, execute_sql_from_jinja_string, get_config
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+
+def deduplicate_taxon(conn, config_parser):
+    import json
```
Comment (damianooldoni) [inline scripts/deduplicate_taxon.py]: Shouldn't this be moved out of the function and be put together with `import os`?

**Review by damianooldoni — APPROVED** [review]
Great work @niconoe!

I agree on the fact that this script should apply permanent changes as this is the way to do: full back compatibility has no sense when we are going to remove duplicates.

Could you update the log file in [changelog](https://github.com/inbo/speciesbim#changelog--development-journal)?

One thought:
It seems you pulled the branch `annexscientificname` in your branch. Unfortunately the code at the stage of pulling is not complete as I was still working on it. Merging to master would bring not reviewed and not working code in master. I am not far from writing a PR for `annexscientificname` branch, but I would like to make you aware of this.

File: scripts/deduplicate_taxon.py (hunk 1 above)
Comment (niconoe) [reply scripts/deduplicate_taxon.py]: Absolutely, it stayed there after a quick and dirty experiment, I'll move it!

**Comment by niconoe** [comment]
Indeed, this branch started from `annexscientificname` and therefore should not be merged to master now.

Since you are close to finishing your work on `annexscientificname`, I suggest that you merge your branch first, once you're ready. Then we'll be able to merge this PR more cleanly and safely (I'll sync the PR by pulling from your branch, if necessary). 

Sorry for the confusion.

**Comment by damianooldoni** [comment]
Great. We set this PR in standby for the moment.

**Pull request merged**